src/moveir/preprocessor/mod.rs

```diff
@@ -350,7 +363,7 @@ impl Visitor for MovePreProcessor {
             }
         }
 
-        if ctx.struct_declaration_context.is_some() && enclosing_identifier != "Quartz_Global" {
+        if ctx.struct_declaration_context.is_some() && enclosing_identifier != "Flint_Global" {
```

Comment:
Bad `is_some`, bring the magic string conditional outside both this and the above condition!